docs/sdd.md

```diff
@@ -8,59 +8,119 @@
 
 ## Overview
 
-This document details the software implementation of FIXME.
+This document details the software implementation of the Arrow telemetry service.
 
-This service is responsible for FIXME
+This service aggregates telemetry transmitted by networked assets (e.g. aircraft, drones, mobile vertiports, etc.) and rebroadcasts to listeners.
+
+It implements a cache to filter out duplicate telemetry reports, such as an aircraft ADS-B message received by multiple networked towers within range.
 
 Attribute | Description
 --- | ---
-Status | Draft
+Status | Development
+Stuckee | A.M. Smith ([@ServiceDog](https://github.com/ServiceDog))
 
 ## Related Documents
 
 Document | Description
 --- | ---
-:construction: High-Level Concept of Operations (CONOPS) | Overview of Arrow microservices.
-:construction: High-Level Interface Control Document (ICD) | Interfaces and frameworks common to all Arrow microservices.
-:construction: Requirements - `svc-FIXME` | Requirements and user stories for this microservice.
-[Concept of Operations - `svc-FIXME`](./conops.md) | Defines the motivation and duties of this microservice.
-[Interface Control Document (ICD) - `svc-FIXME`](./icd.md) | Defines the inputs and outputs of this microservice.
+[High-Level Concept of Operations (CONOPS)](https://github.com/Arrow-air/se-services/blob/develop/docs/conops.md) | Overview of Arrow microservices.
+[High-Level Interface Control Document (ICD)](https://github.com/Arrow-air/se-services/blob/develop/docs/icd.md)  | Interfaces and frameworks common to all Arrow microservices.
+[Requirements - `svc-telemetry`](https://nocodb.arrowair.com/dashboard/#/nc/view/d1bb0a51-e22f-4b91-b1c5-66f11f4f861b) | Requirements and user stories for this microservice.
```

Comment:
Maybe link to the svc-telemetry specific view:
https://nocodb.arrowair.com/dashboard/#/nc/view/6ffa7547-b2ab-4d02-b5cb-ed2d3c60e2c7

Reply:
https://github.com/Arrow-air/svc-telemetry/pull/15/files/0f4d2b06942616bfc605c6c219e6824d3cbd8b66..1ae030ec8c5fc5bf790b7d78560d28c78cdbf4af